src/Native/System.Private.CoreLib.Native/pal_time.cpp

```diff
@@ -33,3 +34,161 @@ mach_timebase_info_data_t *InitializeTimebaseInfo()
     return g_isMachTimebaseInfoInitialized ? &g_machTimebaseInfo : nullptr;
 }
 #endif
+
+extern "C" uint64_t CoreLibNative_GetHighPrecisionCounts()
```

Comment:
Same as for the interop file, could use a singular name since it gets a single count